strictdoc/core/file_traceability_index.py

```diff
@@ -971,6 +1002,11 @@ def create_sdoc_node_from_source_node(
         ):
             sdoc_node_fields["TITLE"] = source_node.entity_name
         FileTraceabilityIndex.set_sdoc_node_fields(sdoc_node, sdoc_node_fields)
+        if "MID" in sdoc_node_fields:
```

Comment:
The same comment as https://github.com/strictdoc-project/strictdoc/pull/2549/files#r2507033876.

Reply:
Moved, same as above.